registers/dump/main.go

```diff
@@ -1,21 +1,69 @@
 package main
 
 import (
+	"context"
+	"encoding/json"
 	"flag"
+	"fmt"
 	"log"
 
 	"github.com/benlaurie/gds-registers/register"
+	"github.com/google/trillian"
+	"google.golang.org/genproto/googleapis/rpc/code"
+	"google.golang.org/grpc"
 )
 
 var (
-	regname = flag.String("register", "register", "name of register (e.g. 'country')")
+	regname     = flag.String("register", "register", "name of register (e.g. 'country')")
+	trillianLog = flag.String("trillian_log", "localhost:8090", "address of the Trillian Log RPC server.")
+	logId       = flag.Int64("log_id", 0, "Trillian LogID to populate.")
```

Comment:
`logID`

Reply:
Not according to Trillian. :-)

Reply:
Unfortunately, the heuristics for capitalising names in protoc generated golang files gets it wrong in the case of acronyms - anything outside of generated code which mixes case for acronyms is wrong and you should tell Martin to fix it :)

Reply:
So, if we call it logID, then we end up with the silliness later of {logId: logID}. I prefer consistency.

Reply:
Style guide is quite explicit about this, even calling out ID as a popular example:
https://github.com/golang/go/wiki/CodeReviewComments#initialisms

:)

Reply:
OK, you win.

Reply:
\o/